modules/cockroachdb/cockroachdb.go

```diff
@@ -233,6 +239,42 @@ func addTLS(ctx context.Context, container testcontainers.Container, opts option
 	return nil
 }
 
+// runStatements runs the configured statements against the CockroachDB container.
+func runStatements(ctx context.Context, container testcontainers.Container, opts options) (err error) { //nolint:nonamedreturns // Needed for error checking.
```

Comment:
bug: this linter directive isn't needed as it was replaced by `nakedret` recently.

Reply:
removed